node/Release/Jenkinsfile

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/groovy
+@Library('github.com/fabric8io/osio-pipeline@master')_
+
+osio {
+
+  config runtime: 'node'
+
+  ci {
+
+    def resources = processTemplate(params: [
+          release_version: "1.0.${env.BUILD_NUMBER}"
+    ])
+
+    build resources: resources, commands: """
```

Comment:
@rupalibehera this still does not address my comments in the earlier version which was to not use `commands` since it gives a **wrong** impression that the commands get executed in build-container